pkg/parsers/kernel/kernel_test_unix.go

```diff
@@ -19,6 +21,7 @@ func assertParseRelease(t *testing.T, release string, b *VersionInfo, result int
 	}
 }
 
+// TestParseRelease tests the ParseRelease() function
```

Comment:
@runcom maybe because the name is `kernel_test_unix.go` now and is not handled properly by golint :stuck_out_tongue_closed_eyes: (or by our `validate-lint` thingy).